spec/requests/api/cost_entries/cost_entries_by_work_package_resource_spec.rb

```diff
@@ -36,8 +36,12 @@
   let(:current_user) {
     FactoryGirl.create(:user, member_in_project: project, member_through_role: role)
   }
-  let(:role) { FactoryGirl.create(:role, permissions: permissions) }
-  let(:permissions) { [:view_cost_entries] }
+  let(:role) { FactoryGirl.create(:role, permissions: total_permissions) }
+  let(:total_permissions) {
+    # always include view WP, to ensure accessibility
```

Comment:
Using the word accessibility here might be misleading, but this should be nothing to worry about.